playbooks/installer/ospd/main.yml

```diff
@@ -1,4 +1,81 @@
 ---
 - include: pre.yml
-- include: run.yml
+
+- name: Installing the undercloud
+  tags: undercloud
+  hosts: undercloud
+  roles:
+      - {role: installer/ospd/undercloud, when: "installer.undercloud is defined"}
+
+- name: Modify overcloud in case of virthost deployment
+  include: virthost.yml
+  tags: virthost
+  when: "'virthost' in groups"
+
+- name: Preparing the images
+  tags: images
+  hosts: undercloud
+  gather_facts: no
+  become: yes
+  become_user: "{{ installer.user.name }}"
+  roles:
+      - {role: installer/ospd/images}
+
+- name: Start introspection
+  tags: introspection
+  hosts: undercloud
+  gather_facts: no
+  become: yes
+  become_user: "{{ installer.user.name }}"
+  roles:
+      - {role: installer/ospd/overcloud/introspection,
+        undercloud_provision_cidr: "{{ ansible_br_ctlplane.ipv4.network }}/{{ ansible_br_ctlplane.ipv4.netmask }}",
+        when: "installer.overcloud is defined"}
+
+- name: Tag hosts with flavors
+  tags: tagging
+  hosts: undercloud
+  gather_facts: no
+  become: yes
+  become_user: "{{ installer.user.name }}"
+  roles:
+      - {role: installer/ospd/overcloud/tagging, when: "installer.overcloud is defined"}
+
+- name: Installing the overcloud
+  tags: overcloud
+  hosts: undercloud
+  become: yes
+  become_user: "{{ installer.user.name }}"
+  pre_tasks:
+      - name: create the template folder for customization
+        shell: "mkdir -p {{ installer.overcloud.template_base }}"
+
+      - name: create the base overcloud deploy script
+        template:
+            src: "templates/overcloud_deploy.sh.j2"
+            dest: "~/overcloud_deploy.sh"
+            mode: 0755
+  roles:
+      - {role: installer/ospd/overcloud/storage, when: "installer.storage is defined"}
+      - {role: installer/ospd/overcloud/network, when: "installer.network is defined and installer.overcloud.network.isolation.enable == 'yes'"}
```

Comment:
According to the spec, the network.isolation propperty will be storred at installer.network.isolation namespace. 

Here is spec snippet: 

```
 network-isolation:
                    type: YamlFile
                    help: The overcloud network isolation type
                    required: yes
network-isolation-template:
                    type: YamlFile
                    help: The overcloud network isolation template

```

So installer.overcloud.network.isolation.enable should be changed to installer.network.isolation.enable